apt_offline_core/AptOfflineCoreLib.py

```diff
@@ -1908,6 +1918,9 @@ def main():
         parser_set.add_argument("--install-src-packages", dest="set_install_src_packages", help="Source Packages that need to be installed",
                           action="store", type=str, nargs='*', metavar="SOURCE PKG")
         
+        parser_set.add_argument("--aptreinstall", dest="set_aptreinstall", help="Generate signatures of packages that already installed in system",
```

Comment:
Please make this more human readable. It anyway is a long option, thus --apt-reinstall will not hurt.